charts/renovate/values.yaml

```
@@ -64,6 +64,10 @@ image:
   # -- "IfNotPresent" to pull the image if no image with the specified tag exists on the node, "Always" to always pull the image or "Never" to try and use pre-pulled images
   pullPolicy: IfNotPresent
 
+slim:
+  # -- Do not add `-slim` suffix to image tag
```

```suggestion
  # -- Add `-slim` suffix to image tag
```
i would prefer to reduce to `slim: false`.

I'll refactor the templates in one of next majors to simplify things.

---

I wanted to align this to the dind configuration. Both could be updated to `slim: true` though.
Changed it to `slim: true` now.